src/card/index.css

```diff
@@ -189,6 +190,7 @@
     border-radius: var(--spectrum-card-quiet-border-radius);
     position: relative;
     transition: background-color var(--spectrum-global-animation-duration-100);
+    overflow: visible;
```

Comment:
Specifically needed here to override something?

Reply:
`.spectrum-Card-preview` was clipping the focus styling.